mteb/tasks/Retrieval/eng/HotpotQARetrieval.py

```diff
@@ -20,7 +20,7 @@ class HotpotQA(AbsTaskRetrieval):
         type="Retrieval",
         category="s2p",
         modalities=["text"],
-        eval_splits=["train", "dev", "test"],
+        eval_splits=["test"],
```

Comment:
I just double checked that this did not cause any problems with existing benchmark

Reply:
It seemed like an error to include the train split in evaluation by default

Reply:
Well some datasets do use the train split in the evaluation as it is the only one

Reply:
right. As far as I know HotPotQA used the test set in the old benchmark though, right?

Reply:
Yes, you're right. HotPotQA is a retrieval dataset and doesn't require train split

Reply:
Yes no problems here